src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int defaultCircleRadius = 10;
```

Comment:
Please use constant naming convention for 'defaultCircleRadius' by making it uppercase and static final: 'DEFAULT_CIRCLE_RADIUS'.